### PR TITLE
Add the pnab package

### PR DESCRIPTION
**Pull request by alenaizan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by alenaizan** [comment]
@conda-forge-admin, please ping team

Ready for review.

File: recipes/pnab/bld.bat [hunk 1]
```diff
@@ -0,0 +1,14 @@
+setlocal EnableDelayedExpansion
+
+CALL "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x86_amd64
```
Comment (ocefpaf) [inline recipes/pnab/bld.bat:3]: @alenaizan I'm not a windows expert but I think conda-build handles that for you.
@isuruf should we remove this line and pin vc14?

Comment (isuruf) [reply recipes/pnab/bld.bat:3]: Yes, this is unneeded.

File: recipes/pnab/bld.bat [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+CALL %BUILD_PREFIX%\Library\bin\cmake -G "NMake Makefiles" -Bbuild -DCMAKE_BUILD_TYPE:STRING=Release -DOPENBABEL_DIR=%PREFIX%
```
Comment (isuruf) [inline recipes/pnab/bld.bat:4]:
```suggestion
CALL %BUILD_PREFIX%\Library\bin\cmake -G "NMake Makefiles" -Bbuild -DCMAKE_BUILD_TYPE:STRING=Release -DOPENBABEL_DIR=%LIBRARY_PREFIX%
```

File: recipes/pnab/bld.bat (hunk 1 above)
Comment (alenaizan) [reply recipes/pnab/bld.bat]: Fixed.

File: recipes/pnab/bld.bat (hunk 2 above)
Comment (alenaizan) [reply recipes/pnab/bld.bat]: Fixed.

**Comment by alenaizan** [comment]
@ocefpaf @isuruf I updated the windows build.

**Pull request merged**